tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # Minimum CMake required
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
```

Comment:
According to https://cmake.org/cmake/help/v3.1/release/3.1.0.html

> New target_compile_features() command allows populating the COMPILE_FEATURES target property, just like any other build variable.

So it should only works from 3.1